api/src/main/resources/liquibase-update-to-2.3.xml

```diff
@@ -108,4 +108,16 @@
 			<column name="description" type="varchar(1)" />
 		</createTable>
 	</changeSet>
+	<changeSet id="TRUNK-5672-201910211105" author="Mritunjay">
```

Comment:
We do not put the changeset in both the 2.3 and 2.4 files. Just one of the two.

Reply:
Sure, it makes sense! I just got confused. I will remove `2.4` and just add it to `2.3`  file

Reply:
Fixed this.